gr-uhd/grc/uhd_fpga_null_src_sink.block.yml

```diff
@@ -0,0 +1,51 @@
+id: uhd_fpga_null_src_sink
+label: RFNoC Null Source/Sink
+
+parameters:
+-   id: type
+    label: RFNoC Block Type
+    dtype: enum
+    default: 'block'
+    options: ['block', 'sep', 'device']
+    hide: all
+-   id: desc
+    label: Block Descriptor
+    dtype: string
+    default: 'null_src_sink.yml'
+    hide: all
+-   id: NIPC
+    label: Items per Clock Cycle
+    dtype: int
+    default: 2
+    hide: part
+-   id: ITEM_W
+    label: Item Width (Bits)
+    dtype: int
+    default: 32
+    hide: part
+
+inputs:
+-   domain: rfnoc.data
+    id: sink
+    dtype: fc32
+    multiplicity: ${ NUM_PORTS }
+-   domain: rfnoc.data
+    id: loop
+    dtype: fc32
+    multiplicity: ${ NUM_PORTS }
+
+outputs:
+-   domain: rfnoc.data
+    id: source
+    dtype: fc32
+    multiplicity: ${ NUM_PORTS }
+-   domain: rfnoc.data
+    id: loop
+    dtype: fc32
+    multiplicity: ${ NUM_PORTS }
+
+documentation: |-
+  RFNoC keep-one-in-n block (FPGA Implementation).
```

Comment:
This is not the correct doc stub for this block.

Reply:
Thanks for catching -- fixed.